src/inmanta/server/protocol.py

```diff
@@ -426,6 +429,9 @@ def __init__(
         self._callhandle: Optional[asyncio.TimerHandle] = None
         self.expired: bool = False
 
+        self.last_dispatched_call: float = 0
+        self.dispatch_delay = 0.1  # keep at least 100 ms between dispatches
```

Comment:
Isn't 100ms a lot? For fast calls it can be a big increase in latency.

Reply:
agreed